kroxylicious-test-tools/src/main/java/io/kroxylicious/test/tester/KroxyliciousClients.java

```diff
@@ -28,13 +30,22 @@
 class KroxyliciousClients {
     private final String bootstrapServers;
 
+    private final List<Admin> admins;
+    private final List<Producer> producers;
+    private final List<Consumer> consumers;
```

Comment:
Could this be a single `List<Closeable> clients`

Reply:
I'd imagine it could, but isn't it better to track them in groups by type so that we have more options later. e.g. adding a `closeProducers` method?